packages/plugins/src/FragmentPlugin.ts

```diff
@@ -0,0 +1,92 @@
+import path from 'path';
+import type { Plugin as PluginType } from '@jpmorganchase/mosaic-types';
+import { escapeRegExp } from 'lodash-es';
+
+interface FragmentPluginPage {
+  fullPath: string;
+  content: string;
+}
+
+const createPageTest = (ignorePages, pageExtensions) => {
```

Comment:
NIT createPageTest is anonymous, you have to read the code to know what the test is 
You could call it 'createHiddenPageFilter' or similar to make the code more descriptive